driver/ppm_flag_helpers.h

```diff
@@ -1781,6 +1781,26 @@ static __always_inline uint32_t linkat_flags_to_scap(int32_t flags)
 	return res;
 }
 
+static __always_inline uint32_t newfstatat_flags_to_scap(int32_t flags)
+{
+	uint32_t res = 0;
+
```

Comment:
do we need an `ifdef` here? if no why?

Reply:
`AT_SYMLINK_NOFOLLOW` was introduced in kernel `2.6.16`, while `AT_EMPTY_PATH` and `AT_NO_AUTOMOUNT` were introduced in kernel `>=2.6.38`
I'm not sure, but I think drivers don't support versions older than `2.6.32`, so we can suppose that `AT_SYMLINK_NOFOLLOW` is always defined.
WDYT?

Reply:
yes, I agree! maybe we can add a comment to remember it